bookmarks/templates/bookmarks/index.html

```diff
@@ -9,7 +9,7 @@
     {# Bookmark list #}
     <section class="content-area col-2">
       <div class="content-area-header mb-0">
-        <h2>Bookmarks</h2>
+        <h2>Bookmarks ({{ bookmark_list.items | length }})</h2>
```

Comment:
`items` only contains the items for the current page, so at max. it would only show the number of items for the current page instead of for the overall search result.